packages/block-library/src/index.native.js

```diff
@@ -151,6 +151,8 @@ export const registerCoreBlocks = () => {
 		// eslint-disable-next-line no-undef
 		!! __DEV__ ? group : null,
 		spacer,
+		// eslint-disable-next-line no-undef
+		!! __DEV__ ? gallery : null,
```

Comment:
Let's remove the DEV flag since this is ready to go. We can add it back if we face with a blocker. The image resolution issue will be fixed on WPiOS side.